GTAIV/README.md

```diff
@@ -45,23 +51,19 @@ LiveSplit Autosplitter for GTA IV + EFLC
 
 Click the "Download Zip" button to download this repository, including autosplitter script, as a .zip file. Use your favourite unarchive utility to unzip the .zip file. -->
 
-## Installation
+<!-- ## Usage
 
 <!-- Add a Scriptable Auto Splitter component to your Layout.
 Right-click on LiveSplit and choose "Edit Layout..." to open the Layout Editor, then click on the Plus-sign and choose "Scriptable Auto Splitter" from the section "Control". You can set the Path of the Script by going into the component settings of the Scriptable Auto Splitter. To get to the settings of the component you can either double click it in the Layout Editor or go into to the Scriptable Auto Splitter Tab of the Layout Settings. Once you've set the Path, the script should automatically load and work. -->
 
-Create or open existing splits and Activate the autosplitter within livesplit, click the settings button to customise the autosplitter for the desired category.
+<!-- Open Livesplit -> right click, click Edit Splits... -> select a game from the top drop-down menu, if you haven't yet -> choose category from drop-down menu below that, if you haven't yet -> click Activate button -> click Settings button -> customize the autosplitter to your likings -->
```

Comment:
Is there any reason why you commented this part out? It seems to be up-to-date to me.